woocommerce/woocommerce
# [CYS] Remove `getLookAndTone` AI request from the business description step

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/41492

We use OpenAI to pre-select the “tone” options, and often, this has failed during the demo. Lauren and Beau have questioned whether this is even necessary for AI to pre-select. And we eventually decided to remove it.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:


1. Create a new WooCommerce installation with this version.
2. Make sure to enable `customize-store` feature flag
3. Go to `WooCommerce -> Home` and click `Customize Store` task
4. Click `Design with AI` button
5. Open dev tools > network tab
6. Fill out the business description
7. Click on "Continue"
8. Observe that no `/text-completion` API requests are sent.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
